tests/installation/bootloader_zkvm.pm

```diff
@@ -56,6 +56,10 @@ sub run() {
             $cmdline .= " autoyast=" . data_url(get_var('AUTOYAST')) . " ";
         }
 
+        if (get_var('FIPS')) {
+            $cmdline .= "fips=1 ";
+        }
```

Comment:
Hmm, it is actually already working. I didn't notice the fips mode is already enabled in specific_bootmenu_params for s390x.